bluesky/ui/qtgl/glhelpers.py

```diff
@@ -35,12 +37,12 @@
 
 
 def get_profile_settings():
-    for version in ((4, 5), (4, 4), (4, 3), (4, 2), (4, 1), (4, 0), (3, 3)):
+    for version in ((4, 1), (2, 1), (2, 0)):
```

Comment:
pyqt6 doesn't seem to support any opengl version after 4.1 see [here](https://www.riverbankcomputing.com/static/Docs/PyQt6/api/qtopengl/qabstractopenglfunctions.html).

Or maybe they don't have documentation for it? 

A bit confused. I could only get it to work with 4.1